### ISSUE TITLE
Redundant text in Wrap-up quiz 7

### ISSUE THREAD

**Issue by ArturoAmorQ** [INRIA/scikit-learn-mooc#613]
Prelude to Q6 says:

> In the previous cross-validation, we made the choice of using a `ShuffleSplit`
cross-validation strategy. It means that randomly selected samples were selected
as testing set ignoring any time dependency between the lines of the dataframe.
> We would like to have a cross-validation strategy that evaluates the capacity of
our model to predict on a completely new bike ride: the samples in the
validation set should only come from rides not present in the training set.

Prelude to Q7 says:

> Instead of using the naive `ShuffleSplit` strategy, we will use a strategy that
takes into account the group defined by each individual date. It corresponds to
a bike ride. We would like to have a cross-validation strategy that evaluates
the capacity of our model to predict on a completely new bike ride: the samples
in the validation set should only come from rides not present in the training
set.

As the text is more inline with Q7, I suggest removing the prelude to Q6.

**This should be done once the session 2 has finished**.

**Comment by ArturoAmorQ** [comment]
Addressed in [!72](https://gitlab.inria.fr/learninglab/mooc-scikit-learn/mooc-scikit-learn-coordination/-/merge_requests/72) (not yet merged).

**Comment by lesteve** [comment]
This has been fixed.
